lib/dry/logic/predicates.rb

```diff
@@ -134,7 +134,15 @@ def self.included(other)
       end
 
       predicate(:exclusion?) do |list, input|
-        !self[:inclusion?].(list, input)
+        !list.include?(input)
```

Comment:
What was your thinking behind the adjustment here, @fran-worley? Seemed like the approach of taking the `inclusion?` predicate and inverting the result was quite neat (and perhaps something we could do for `excludes?` below?